src/test/java/api/loans/CheckOutByBarcodeTests.java

```diff
@@ -920,4 +921,241 @@ public void cannotCheckOutWhenItemIsNotLoanable()
       hasItemBarcodeParameter(nod),
       hasLoanPolicyParameters(notLoanablePolicy))));
   }
+
+  @Test
+  public void canCheckOutWhenItemLimitOfLoanPolicyIsReachedButMaterialTypeAndLoanTypeMissing()
```

Comment:
What are loan type and material type missing from?  Is this the scenario where there is no material type or loan type defined in the rules?

Reply:
yes, that's right. This is the scenario where there is no material type or loan type defined in the rule.

Reply:
Please could this test be renamed to make that intent clearer.

How can an item limit that uses a count based upon material type or loan type be reached when neither are provided?

Reply:
Since material type and loan type are not present in circulation rule, according to the scenario 8 and 9 of the story [CIRC-558](https://issues.folio.org/browse/CIRC-558) the item limit should be ignored and next item successfully checked-out

Reply:
Maybe this test could be reworded as something like `canCheckOutWhenItemLimitIsIgnoredForRulesWithoutMaterialTypeOrLoanType`?

Reply:
renamed